src/api/lib/crypto/crypto.rb

```diff
@@ -0,0 +1,34 @@
+module Crypto
+  module Encrypt
+    def self.cloud_upload_data(data)
+      crypto = GPGME::Crypto.new(armor: true)
+      crypto.encrypt(data, recipients: PublicKeys.cloud_upload.fingerprint, always_trust: true).to_s
```

Comment:
What if this goes wrong? Your application ID will the be the error as string?

Reply:
According to the gem, it will raise either and `GPGME::Error::InvalidValue` or `GPGME::Error::UnusablePublicKey`

Reply:
Okay then it won't be. Do do you think you need to catch the exceptions?

Reply:
Hm afer looking at this more closely because of the crash you are seeing, what is `PublicKeys.cloud_upload.fingerprint`? Is `.fingerprint` something from the `gpgme`?

Reply:
@hennevogel well, I don't think we need to catch exceptions here. If it doesn't work there is something very very wrong.

And yes, `.fingerprint` comes from `gpgme`.